examples/chat/chat.go

```diff
@@ -129,6 +128,22 @@ func (cs *chatServer) subscribe(ctx context.Context, c *websocket.Conn) error {
 	cs.addSubscriber(s)
 	defer cs.deleteSubscriber(s)
 
+	go func() {
```

Comment:
without the `ctx` from `closeread`, you'll have to make your own or `defer close(s.msgs)` here.